projects/schematics/src/add-spartacus/index.ts

```diff
@@ -488,6 +491,42 @@ function updateAppModule(options: SpartacusOptions): Rule {
   };
 }
 
+/**
+ * Adds `import { AppRoutingModule } from "@spartacus/storefront"` to the given app.module sourceFile.
+ *
+ * If a local file with the same module name `AppRoutingModule` already exists in the project,
+ * it will be removed first, and later the `AppRoutingModule` from `@spartacus/storefront` will be imported.
+ *
+ * Note: Since v17 Angular `ng new` command by default creates a local `AppRoutingModule` file in the project.
+ *       So we have to replace it.
+ *
+ * See Angular enabling routing by default in v17: https://github.com/angular/angular-cli/commit/1a6a139aaf8d5a6947b399bbbd48bbfd9e52372c
+ */
+function addAppRoutingModuleImport(tree: Tree, sourceFile: SourceFile) {
+  // remove import of AppRoutingModule (NgModule import and module path import), if exists
+  removeModuleImport(sourceFile, {
```

Comment:
`nitpick:` I would consider whether log here that the import path has been changed, in favor of debug mode.